react/features/toolbox/functions.any.ts

```diff
@@ -19,12 +20,18 @@ export function isAudioMuteButtonDisabled(state: IReduxState) {
 /**
  * Returns the buttons corresponding to features disabled through jwt.
  *
- * @param {IReduxState} state - The state from the Redux store.
+ * @param {string | undefined} jwt - The jwt token.
+ * @param {ILocalParticipant} localParticipantFeatures - The features of the local participant.
  * @returns {string[]} - The disabled by jwt buttons array.
  */
-export function getJwtDisabledButtons(state: IReduxState) {
+export function getJwtDisabledButtons(jwt: string | undefined, localParticipantFeatures?: IParticipantFeatures) {
```

Comment:
Can you please explain how this change accomplished the set goal, since AFAIS it still returns a different array on every call.